cli/debugging-issues.adoc

```diff
@@ -15,7 +15,21 @@ include::modules/viewing-the-logs.adoc[leveloffset=+1]
 
 include::modules/viewing-current-log-level.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../cli/command-reference/roxctl-central.adoc#roxctl-central-debug_roxctl-central[roxctl central debug]
+
 include::modules/changing-the-log-level.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../cli/command-reference/roxctl-central.adoc#roxctl-central-debug_roxctl-central[roxctl central debug]
+
 include::modules/retrieving-debugging-information.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../cli/command-reference/roxctl-central.adoc#roxctl-central-debug_roxctl-central[roxctl central debug]
```

Comment:
![image](https://github.com/openshift/openshift-docs/assets/1616386/b25537fe-3ba4-42d6-85fc-26c689d45627)
I think it's a bit too much

Reply:
Unfortunately, this is the only option we have as the "Additional resources" section on "roxctl central debug command options" is not applicable to the entire assembly. Otherwise, we could have created a level 2 heading for "Additional resources" that applies to the entire assembly. 

@kcarmichael08 Please chime in on this one? Thx!